src/platform/datapath_winuser.c

```diff
@@ -577,15 +577,14 @@ QuicDataPathInitialize(
         goto Exit;
     }
 
-    GetSystemInfo(&Sysinfo);
     DatapathLength =
         sizeof(QUIC_DATAPATH) +
-        Sysinfo.dwNumberOfProcessors * sizeof(QUIC_DATAPATH_PROC_CONTEXT);
+        MaxProcCount * sizeof(QUIC_DATAPATH_PROC_CONTEXT);
 
     //
     // Thread affinity uses a bit mask. We can't support more cores than bits.
     //
-    QUIC_FRE_ASSERT(Sysinfo.dwNumberOfProcessors <= sizeof(DWORD_PTR) * 8);
+    QUIC_FRE_ASSERT(MaxProcCount <= sizeof(DWORD_PTR) * 8);
```

Comment:
Wouldn't this assertion still fail on more then 64 cores?

Reply:
Fixed.